design-reviews/recipes/engagement-process.md

```diff
@@ -0,0 +1,129 @@
+
+## Game Plan
+
+The game plan document is our guiding star throughout the engagement.  The game plan document should include the following sections and answer the following questions.
+
+### Deliverable
+
+1. What are we delivering?
+1. What does success look like?
+1. What is the definition of done?
+1. What security, privacy and/or performance requirements are required?
+
+### Design
+
+1. What does the high level architecture look like?
+1. What does the technology stack look like?
+1. How does this solution fit into the customers overall product / service?
+
+### Process
+
+1. How will engineering fundamentals be leveraged during the project?
+1. What is the proposed sprint schedule?  How many sprints?
+1. Who is the team? CSE engineers? Customer Engineers?
+1. What is the sharing plan after the engagement is complete?
+
+### Ready to start
+
+1. Has the customer signed off on the plan?
+1. Are there any blocking issues to starting the engagement?
+1. What are the known risks?
+
+> **Tip**: Keep your game plan up to date as your design changes over time
+
+## Checkpoint Reviews
+
+Our monthly checkpoints are a great way for the team to check-in with leadership and other CSE SME's and provide a status update to the larger team.
+
+Some key items to discuss:
+
+### Design Reviews
+
+1. Present any key design reviews created since the last checkpoint or game plan
+1. Present any new design artifacts including API contracts, architecture diagrams, workflows, deployment pipelines, etc.
+1. Describe if there are any high level architecture or design changes since last review
+1. How have your design changes impacted the initial game plan
+
+## Sprint Planning
+
+In many of our engagements we will be working with our customers using a SCRUM agile development process.  Sprint planning is a great opportunity to dive deep into the next set of high priority work.  Some key points to address are the following:
+
+1. Identify stories that require design reviews
+1. Breakout design from implementation for complex stories
+1. Assign owners to each design story
+
+Stories that will benefit from design reviews have one or more of the following in common:
+
+1. Are there many unknowns or unclear requirements
+1. There is wide distribution in story point costing amongst your dev crew
+1. You cannot clearly illustrate all the tasks required for a story
+
+> **Tip:** After your sprint planning is complete your team should consider hosting an initial design review discussion to dive deep on the design requirement of the stories that were identified.  After you have more clarity the team should continue forward with either a sync or async design review to complete the tasks.
+
+## Sprint Backlog Grooming
+
+If you team is not already hosting a Sprint Backlog Grooming session at least once per week you should consider it.  It is a great opportunity to:

```

Comment:
Maybe linking this to https://github.com/microsoft/code-with-engineering-playbook/tree/master/backlog-management